### ISSUE TITLE
Versioning in HABpanel widgets

### ISSUE THREAD

**Issue by BasvanH** [openhab/org.openhab.ui.habpanel#296]
It would be handy to add versioning with HABpanel widgets, and perhaps some form of notification when a widget has been updated by the author.

**Comment by ghys** [comment]
Okay, seems like a nice idea. 🙂
That would involve the widget author maintaining the versions, though and it should avoid hitting the GitHub API too much. The version field should remain optional and offer update checks for versioned widgets only (some authors won't care about versions).
Also maybe it should offer a link to the readme/forum post when updating since simply updating the widget is often not enough (additional files in `conf/html` would need to be updated too) and there hopefully will be instructions there.

**Comment by BasvanH** [comment]
Yes, you are correct. When updating the widget some dependencies may change also. We can start with an initial version, and if people don't want to update it, leave at this init one.
